src/web_node.ml

```diff
@@ -90,7 +92,7 @@ let addEventListener n typ listener options = n##addEventListener typ listener o
 
 let removeEventListener n typ listener options = n##removeEventListener typ listener options
 
-let focus n = n##focus ()
```

Comment:
Why are there random empty tuples being added to some calls?  o.O?
They do not have optional arguments or anything...

Reply:
This one is definitely my fault. It should be you did it and my type signature in the mli should be changed to `t -> unit`.